### DISCUSSION TITLE
AlmaLinux version is stuck in 276

### DISCUSSION THREAD

**Discussion by ghnp5** [cockpit-project/cockpit#18815] [Q&A] [answered]
Hey!

AlmaLinux 8's cockpit version is currently 276.

Is this because newer versions don't support AlmaLinux 8, or is it possible to update the version available in the "baseos"/"appstream" repositories?

Thank you very much!

**Comment by martinpitt** [comment, marked as answer]
We update CentOS 8 Stream with recent Cockpit versions all the time. E.g. just yesterday I updated it to the very latest version 292. I don't know how Alma works and when/how they update. So it may be worth sending a bug report to them asking for importing a new version from CentOS.
